docs/preview/yamlgettingstarted-resources.md

```diff
@@ -0,0 +1,60 @@
+# YAML getting started - Resources (coming in April)
+
+Resources enable the definition of various items which are to be produced or consumed
+by the defining pipeline. 
+
+## Repositories
+
+The basic definition of a repository is something similar to below. The `alias` and 
+`type` are the only properties shared among all repository types. 
+
+```yaml
+# File: .vsts-ci.yml
```

Comment:
nit: I would remove this line. I had included file names in the examples for templates to try and help distinguish between the different files involved in the example. Otherwise none of the examples in our docs include file names.